tools/remark-cli/package-lock.json

```diff
@@ -2,7 +2,6 @@
   "name": "remark-cli",
   "version": "4.0.0",
   "lockfileVersion": 1,
-  "preserveSymlinks": "1",
```

Comment:
Should not be checked in.

Kinda annoying `make lint` still touches this file, I thought that had been fixed.